variancetransform.py

```diff
@@ -0,0 +1,151 @@
+__doc__ ="""\
+VarianceTransform
+=================
+
+**VarianceTransform** 
+
+This module allows you to calculate the variance of an image, using a determined window size. It also has
+the option to find the optimal window size from a predetermied range to obtain the maximum variance of an image.
+
+============ ============ ===============
+Supports 2D? Supports 3D? Respects masks?
+============ ============ ===============
+YES          NO           NO
```

Comment:
1) Are we sure respects masks is no here?
2) I would think it would be pretty easy to at least allow slicewise 3D - what do you think?